### DISCUSSION TITLE
Enhance DapsExtension to Validate referringConnector Claim

### DISCUSSION THREAD

**Discussion by efiege** [eclipse-edc/Connector#2413] [Ideas]
It would be great if there is a `@Setting` which is compared to the referringConnector claim of a DAT to validate, that the DAT has been issued correctly by the DAPS.

What do you think? Could we implement that feature?

**Comment by ndr-brt** [comment]
Hi, I think it already exist, please checkout the [ids-validation-token extension](https://github.com/eclipse-edc/Connector/tree/main/data-protocols/ids/ids-token-validation), `edc.ids.validation.referringconnector` should be what you are looking for

**Reply by efiege** [reply]
Hey, 
this extension ensures, that the incoming IDS-Messages `issuerConnector` is equal to the `referringConnector` in the DAT. The idea behind this discussion was to verify that the outgoing DAT was correctly issued for a perticular connector instance.

**Comment by gbrost** [comment]
Might be interesting for you, @redingjl ?

**Reply by jimmarino** [reply]
FYI, sending outgoing certificates has been removed from the Dataspace Protocol Specifications and the EDC due to a potential security exploit.
